src/partials/footer.handlebars

```diff
@@ -4,7 +4,7 @@
       <p>
         Copyright &copy; <span id="copyright-year">2015</span> <a href="http://apache.org">The Apache Software Foundation</a>,
         Licensed under the Apache License, Version 2.0<br>
-        Apache and the Apache feather logo are trademarks of The Apache Software Foundation.<br>
+        Apache and the Apache feather logo are trademarks of The Apache Software Foundation. | <a href="/privacy.html">Privacy Policy</a><br>
```

Comment:
Ok I will go with it. Thought relative would be better but as you indicated the absolute path seems to be prevalent. We can address these collectively later.